src/utils/use_window_size.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from "react"
+
+export default function useWindowSize() {
+  const isClient = typeof window === "object"
+
+  function getSize() {
```

Comment:
Why not a simple guard returning `{}` if `isClient` is falsy?

Reply:
save as above